.travis.yml

```diff
@@ -10,6 +10,9 @@ install:
 services:
  - postgresql
 
+
```

Comment:
Hey Nick you have a repetition of services

Reply:
got that, thanks